contracts/mixnet/src/transactions.rs

```diff
@@ -1511,7 +1667,8 @@ pub mod tests {
 
         let initial_bond = 100_000000;
```

Comment:
Maybe change this to `100_000_000` for readability

Reply:
I've actually left it this as `100_000000` for readability as in the contract we operate on the micro-currencies. Hence this would imply `100` of the currency "proper". But I don't mind changing it if you think it'd be better